3.0-RELEASE-NOTES.md

```diff
@@ -5,3 +5,14 @@ always describe the impact on users and instructions for upgrading
 applications from 2.x to 3.0.
 
 See also https://github.com/strongloop/loopback/blob/master/3.0-DEVELOPING.md
+
+
+## API call for remote method has been changed.
```

Comment:
I find this little bit confusing, I would expect this is changing the way how remote methods are invoked. Instead, we are changing the API for searching/looking up a remote method by name. Could you please rephrase?